guides/concepts/networks.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Networks
+---
+
+Stellar has two public networks: the Public Network (pubnet), which is the main decentralized
+network used throughout the world, and the Test Network (testnet), a network maintained by the
+Stellar Development Foundation that is available for developers to test their applications. For
+more in-depth information on testnet, [see our our additional documentation](test-net.md).
+
+## Network Passphrases
+
+Each Stellar network has its own unique passphrase, which is used when validating signatures on a
+given transaction. If you sign a transaction for one network but submit it to another, it won't be
+considered valid. The format of a passphrase is typically `'[Network Name] ; [Month of Creation]
+[Year of Creation]'`.
+
+The current passphrases for the Stellar pubnet and testnet are:
+
+* Pubnet: `'Public Global Stellar Network ; September 2015'`
+* Testnet: `'Test SDF Network ; September 2015'`
+
+The passphrase is utilized for two reasons:
+
+* The network passphrase is used as the seed for the root account (master network key) at genesis.
```

Comment:
Since you just said "is utilized," I'd get rid of "is used," and keep the construction parallel to the second bullet point.

My suggestion:

The passphrase serves two main purposes:
* It's used as the seed for the root account...
* It's used to build hashes of transactions...